# Combined system info message

Header          header

string          hostname

BatteryStatus   battery
NetworkStatus   network
SystemStatus    system
